Reply/app/src/main/res/values/color.xml

```diff
@@ -14,34 +14,64 @@
   -->
 <resources>
 
```

Comment:
Can you rename these to all use snake case instead of a mix of camel and snake?

e.g 
reply_light_onPrimary -> reply_light_on_primary